src/TodoListItem.js

```diff
@@ -1,7 +1,18 @@
 import React from "react";
 
-const TodoListItem = ({ item }) => {
-  return <li>{item.title}</li>;
+const TodoListItem = ({ id, title, onRemoveTodo }) => {
+  const handleRemoveClick = () => {
+    onRemoveTodo(id);
+  };
+
+  return (
+    <div>
+      <span>{title}</span>
+      <button type="button" onClick={handleRemoveClick}>
+        Remove
+      </button>
+    </div>
```

Comment:
This still needs to be a `<li>` and you don't need that <span>. There's nothing technically wrong with your `handleRemoveClick` function, but since all it's doing is calling another function you can do the same thing by passing an anonymous function into `onClick`

```suggestion
    <li>
      {todo.title}
      <button type="button" onClick={() => onRemoveTodo(todo.id}>
        Remove
      </button>
    </li>
```